src/hooks/useAnalyticsTokenInput.ts

```
@@ -0,0 +1,262 @@
+          timestamp: Date.now(),
+        })
+        
+        lastAmountRef.current = amount
+      } catch (error) {
+        console.error('Analytics: Failed to track amount change:', error)
+      }
+    }, debounceMs)
+  }, [enabled, trackAmountChanges, debounceMs])
+
+  // Track input focus
+  const trackInputFocus = useCallback(async (inputType: 'amount' | 'token', tokenSymbol?: string) => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'trackInputFocus'. Consider to delete it or rename it.

'trackInputFocus' is defined here:


(lint/suspicious/noRedeclare)